src/install/ai-monitoring/agent-lang/go-config.mdx

```diff
@@ -0,0 +1,70 @@
+---
+componentType: default
+headingText: Configure the Go agent
+freshnessValidatedDate: never
+---
+
+To collect AI data, you need to ensure that certain configurations are enabled when you instrument your AI-powered app. We also recommend updating other variables to improve data retention.
+
+* We recommend applying these configurations in the New Relic config file. 
+* If you have a more complex set up with multiple environments, you can configure with environment variables.
+
+Set these variables:
```

Comment:
The table is a weird way to present this info. just scanning this, i'm unsure if I should enable them in all 3 ways and how to do that at all.

I'd move each category into a collapser with slightly more detailed instructions